.golangci.example.yml

```diff
@@ -550,6 +550,15 @@ linters-settings:
     multi-if: false   # Enforces newlines (or comments) after every multi-line if statement
     multi-func: false # Enforces newlines (or comments) after every multi-line function signature
 
+  wrapcheck:
+    ignoreSigs:
```

Comment:
```suggestion
    # An array of strings that specify substrings of signatures to ignore.
    # If this set, it will override the default set of ignored signatures.
    ignoreSigs:
```